APIGateway/.env

```diff
@@ -1,9 +1,9 @@
-METRICS_DEV_FOCUS_URL=http://localhost:8001
-METRICS_BURNDOWN_URL=http://localhost:8001
-METRICS_LEADTIME_URL=http://localhost:8001
-METRICS_CYCLETIME_URL=http://localhost:8001
-METRICS_CRUFT_URL=http://localhost:8001
+METRICS_DEV_FOCUS_URL=http://backend:8001
```

Comment:
How are we going to do the review? Running from docker-compose would be difficult.

Reply:
We'll have to make changes in .env to test, but I think our application should be ready to run on docker compose